courses/.htaccess

```diff
@@ -0,0 +1,5 @@
+Options +FollowSymLinks
+RewriteEngine on
+
+# Match ontology
+RewriteRule ^coursesonto(.*)$ https://github.com/naiayti/TeachingKG/blob/main/ontology/ontology.owl [R=303,L]
```

Comment:
Can you confirm that you want urls like `https://w3id.org/courses/coursesonto...`?  Asking since people sometimes duplicate the prefix.  It's different here, but thought I'd ask.